pkg/codegen/schema.go

```diff
@@ -231,8 +231,12 @@ func GenerateGoSchema(sref *openapi3.SchemaRef, path []string) (Schema, error) {
 			// We default to int if format doesn't ask for something else.
 			if f == "int64" {
 				outSchema.GoType = "int64"
+			} else if f == "uint64" {
```

Comment:
This is not valid swagger. uint64 and uint32 are not supported, see "Data Types" here:
https://swagger.io/specification/

I wonder if we could do this with x-go-type

Reply:
@deepmap-marcinr it will be great if we can do it any how, because my current `go-pg` schema `id` are `uint64`

Reply:
Well, it makes us more permissive, we still work with vanilla swagger, so ok.